tools/cockpit.spec

```diff
@@ -93,6 +93,7 @@ Recommends: %{name}-docker = %{version}-%{release}
 Suggests: %{name}-pcp = %{version}-%{release}
 Suggests: %{name}-kubernetes = %{version}-%{release}
 Suggests: %{name}-selinux = %{version}-%{release}
+Suggests: %{name}-pkgupdates = %{version}-%{release}
```

Comment:
Right, ostree's manifest name is "updates", so it should probably have been called "ostree-updates", and we could call this "packagekit-updates" or "pk-updates" for brevity. To compare, we wouldn't put the functionality of searching and installing new apps (what @mvollmer is working on) into ostree just because that's also going to work on ostree.

Naming packages entirely after the underlying interfaces that they use is tricky -- for example, lots of our code talks to systemd,  not only pkg/systemd.

So, I'm not sure about the package name for this, I just have a strong feeling that "cockpit-packagekit" is really misleading and ambiguous, and it should be something else. Meeting topic?